components/Button/utils.ts

```diff
@@ -0,0 +1,14 @@
+export enum ButtonType {
+  PRIMARY = "primary",
+  SECONDARY = "secondary",
+  NAVIGATION = "navigation",
+}
+
+export const defaultClassName = {
```

Comment:
probably it will be better to use defaultButtonClassName - this naming will be more readable in other files